mitodl/bootcamp-ecommerce
# Add redux-query and update API logic to use it

#### What are the relevant tickets?
Fixes #413 

#### What's this PR do?
Adds redux-query and migrates existing API requests to use that library

#### How should this be manually tested?
Nothing should break. Specificially, you should be able to make a purchase and things should work out correctly. You don't need to do order fulfillment for this since this is only front-end work.

Let me know if you want to test this on the PR build. Some oauth settings need tweaking in that case but otherwise it should be straightforward.
